Remotion/ObjectBinding/Web/UI/Controls/Validation/BocListValidationResultDispatchingValidator.cs

```diff
@@ -44,6 +50,19 @@ public void DispatchValidationFailures (IBusinessObjectValidationResult validati
 
       if (dispatchToValidationFailureRepository)
         DispatchToValidationFailureRepository(validationResult, bocListControl);
+
+      var rowObjects = bocListControl.Value?.Cast<IBusinessObject>() ?? Array.Empty<IBusinessObject>();
+
+      _numberOfNotVisibleValidationErrors = rowObjects.Select(r => validationResult.GetUnhandledValidationFailures(r).Count).Sum();
+
+      var control = GetControlToValidate();
+
+      _validationFailuresOfList = validationResult.GetValidationFailures(control, markAsHandled: true).ToArray();
```

Comment:
Ah, okay, those are the validation failures directly associated with the Property bound to BocList, not the failures found in the individual rows.

Maybe `_validationFailuresForBusinssObjectPropertyOfBocList`
And thus `_numberOfNotVisibleValidationErrors` => `_numberOfValidationFailuresForBocListRowsInvisibleDueToPagination`

A mouthful, but precise and selfdocumenting.